hazelcast/src/main/java/com/hazelcast/jet/impl/JetClientInstanceImpl.java

```diff
@@ -126,6 +141,75 @@ public Job newJobProxy(long jobId, boolean isLightJob, @Nonnull Object jobDefini
         return new ClientJobProxy(client, jobId, isLightJob, jobDefinition, config);
     }
 
+    @Override
+    public void submitJobJar(@Nonnull Path jarPath, String snapshotName, String jobName, String mainClass,
+                             @Nonnull List<String> jobParameters) {
+
+        try {
+            UUID sessionId = UuidUtil.newSecureUUID();
+
+            String fileName = fileNameWithoutExtension(jarPath);
+            String sha256Hex = calculateSha256Hex(jarPath);
+            long jarSize = Files.size(jarPath);
+
+            // Send job meta data
+            boolean result = sendJobMetaData(sessionId, fileName, sha256Hex, snapshotName, jobName, mainClass,
+                    jobParameters);
+            if (result) {
+                logFine(getLogger(), "Submitted JobMetaData successfully for jarPath: %s", jarPath);
+            }
+            // Send job parts
+            sendJobMultipart(jarPath, sessionId, jarSize);
+
+        } catch (IOException | NoSuchAlgorithmException exception) {
+            sneakyThrow(exception);
```

Comment:
```suggestion
            throw sneakyThrow(exception);
```

Reply:
sneakyThow alread throw. There is no need to add a new throw keyword